internal/cli/kraft/run/run.go

```diff
@@ -51,7 +51,7 @@ type RunOptions struct {
 	WithKernelDbg bool     `long:"symbolic" usage:"Use the debuggable (symbolic) unikernel"`
 
 	workdir           string
-	platform          mplatform.Platform
+	Platform          mplatform.Platform `noattribute:"true"`
```

Comment:
Can you move this up into the other "exported" fields and sort alphabetically? :pray: